packages/js/client/src/PolywrapClient.ts

```diff
@@ -202,7 +215,7 @@ export class PolywrapClient implements Client {
   >(options: QueryOptions<TVariables, TUri>): Promise<QueryResult<TData>> {
     let result: QueryResult<TData>;
 
-    try {
+    err: try {
```

Comment:
Just curious here: What's the reason for this syntax? I am noticing that you are then using a break but we can also return `{ errors: [parseResult.error as Error] }`? Probably I am wrong but just wanted to understand this :P

Reply:
It's because of the `clearContext` below the try catch statement. I wanted to be sure it still ran since it was running on error before the changes.